lib/wallet/api/http/Cardano/Wallet/Api/Http/Server/Error.hs

```diff
@@ -769,6 +769,11 @@ instance IsServerError ErrDerivePublicKey where
 instance IsServerError ErrAddCosignerKey where
     toServerError = \case
         ErrAddCosignerKeyNoSuchWallet e -> toServerError e
+        ErrAddCosignerKeyNoMeta ->
+            apiError err503 NoMeta $ mconcat
+                [ "No meta of wallet was found in database during migration"
```

Comment:
@paweljakubas 

When you refer to "no meta of wallet", do you mean "no wallet metadata"?

Reply:
yes